.github/workflows/build-windows.yml

```diff
@@ -311,6 +311,8 @@ jobs:
           ./shards ../src/tests/gfx_cube.edn
           echo "Running test: gfx_materials"
           ./shards ../src/tests/gfx_materials.edn
+          echo "Running test: ui_tests"
+          ./shards ../src/tests/ui_tests.edn
```

Comment:
Could just call it ui.edn or egui.edn since it's already inside the tests folder (we don't name the other files in this folder _tests either)

Reply:
I was looking at this file as a collection of unit tests. Compared to the `egui_demo.edn` which is more like a complete sample.

Reply:
I can rename it `ui_unit_tests.edn`. @guusw What do you think?